comprehension1.py

```diff
@@ -0,0 +1,18 @@
+sarah_fav_stuff = [17, 'Sophia', 46.7, 'piano', 'April 21, 2023']
+
+# collect all the string values in 'stuff'
+strs = [strings for strings in sarah_fav_stuff if isinstance(strings, str)]
+
+print(strs)
+# collect all the number values in 'sarah_fav_stuff'
```

Comment:
Remove commented lines that aren't actual comments

Reply:
removed